src/lib/ngsi/StringList.cpp

```diff
@@ -170,23 +170,23 @@ void AttributeList::push_back(const std::string& attributeName)
 *
 * push_back_if_absent - 
 */
-void AttributeList::push_back_if_absent(const std::string& attributeName)
+void StringList::push_back_if_absent(const std::string& string)
 {
-  if (lookup(attributeName) == false)
+  if (lookup(string) == false)
   {
-    attributeV.push_back(attributeName);
+	  stringV.push_back(string);
```

Comment:
Style: excessive whitespace (it should be 2 spaces)

Reply:
Fixed in be6a06c